### PR TITLE
Lazy-load the CharmEditor

### PR DESCRIPTION
**Pull request by mattcasey**

### CODE REVIEW

**Comment by Devorein** [comment]
image
Creating a bounty throws this error

File: components/common/IntlProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ReactNode, useEffect } from 'react';
+import { IntlProvider } from 'react-intl';
+import { useAppDispatch, useAppSelector } from 'components/common/BoardEditor/focalboard/src/store/hooks';
+import { getMessages } from 'components/common/BoardEditor/focalboard/src/i18n';
+import { fetchLanguage, getLanguage } from 'components/common/BoardEditor/focalboard/src/store/language';
+
+export default function ReactDndProvider ({ children }: { children: ReactNode }) {
```
Comment (Devorein) [inline components/common/IntlProvider.tsx:7]: Not sure why its called ReactDndProvider 🤔

**Review by Devorein — CHANGES_REQUESTED** [review]

**Comment by Devorein** [comment]
Tested all the major operations of the platform to make sure nothing broke. Thankfully caught one. Other than that everything looks and works great. We can merge the PR once we handle that case

**Comment by Devorein** [comment]
I've fixed both of the issues in the latest commits. I will do one more round of QA to make sure everything is a-okay

**Review by Devorein — APPROVED** [review]

File: components/common/IntlProvider.tsx (hunk 1 above)
Comment (mattcasey) [reply components/common/IntlProvider.tsx:7]: mistake!

**Pull request merged**